ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/Leaf.java

```diff
@@ -74,48 +73,49 @@ public String getMessage() {
      *
      * It is built directly from a RaoData on which a systematic sensitivity analysis could hav already been run or not.
      */
-    // This constructor is useful to mock SystematicSensitivityComputation in LeafTest
-    Leaf(RaoData raoData, RaoParameters raoParameters, SystematicSensitivityComputation systematicSensitivityComputation, IteratingLinearOptimizer iteratingLinearOptimizer) {
+    // This constructor is useful to mock SystematicSensitivityInterface in LeafTest
```

Comment:
is it still true ? SystematicSensitivityInterface is instantiated within this method

Reply:
indeed! the constructor is used to build the root leaf! removed the erroneous comment